src/api/app/models/token.rb

```diff
@@ -7,6 +7,7 @@ class Token < ApplicationRecord
   has_secure_token :string
 
   validates :user, presence: true
+  validates_uniqueness_of :string, case_sensitive: false
```

Comment:
To pass the RuboCop checks:

```suggestion
  validates :string, uniqueness: { case_sensitive: false }
```

I guess this is a tiny bit better since we can group up column validations on a single line. I don't have a strong opinion on this one.

Reply:
Done, thanks, somehow overlooked that rubocop complains....